src/main/java/domain/Rank.java

```diff
@@ -0,0 +1,45 @@
+package domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(6, new Money("2000000000")),
+    SECOND(5, new Money("30000000")),
+    THIRD(5, new Money("1500000")),
+    FOURTH(4, new Money("50000")),
+    FIFTH(3, new Money("5000")),
+    NOTHING(0, new Money("0"));
+
+    private static final long FIVE = 5;
+
+    long count;
+    Money reward;
```

Comment:
```suggestion
    private final long count;
    private final Money reward;
```
`count`보다 더 좋은 이름이 있지 않을까요?

Reply:
[`fbbb2af`](https://github.com/woowacourse/java-lotto/pull/248/commits/fbbb2af55c7aefb636b20b2e03b3ef87e6fec500) 수정했습니다!